### PR TITLE
Error if no sessions decrypted

### PR DESCRIPTION
**Pull request by dbkr**
Tell the user if they entered the wrong recovery key

Fixes https://github.com/vector-im/riot-web/issues/8143

### CODE REVIEW

File: src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js [hunk 1]
```diff
@@ -184,6 +184,15 @@ export default React.createClass({
         } else if (this.state.backupInfo === null) {
             title = _t("Error");
             content = _t("No backup found!");
+        } else if (this.state.recoverInfo && this.state.recoverInfo.imported === 0) {
+            title = _t("Error Restoring Backup");
+            let failedToDecrypt;
+            content = <div>
+                <p>{_t(
+                    "Failed to decrypt any sessions: check that the recovery key " +
```
Comment (jryans) [inline src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js]: Hmm, this string feels a bit too technical too me... Perhaps something like: "Riot wasn't able to decrypt anything from your key backup. Please verify that you entered the correct recovery key."

Comment (dbkr) [reply src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js]: yeah, good point - have simplified the first bit further to, "failed to restore backup".

File: src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js [hunk 2]
```diff
@@ -184,6 +184,14 @@ export default React.createClass({
         } else if (this.state.backupInfo === null) {
             title = _t("Error");
             content = _t("No backup found!");
+        } else if (this.state.recoverInfo && this.state.recoverInfo.imported === 0) {
+            title = _t("Error Restoring Backup");
+            let failedToDecrypt;
+            content = <div>
+                <p>{_t(
+                    "Backup restore failed: please verify that you entered the correct recovery key.",
```
Comment (jryans) [inline src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js]: Does the "Backup restore failed: " prefix seem redundant given the dialog title "Error Restoring Backup"?

Comment (dbkr) [reply src/components/views/dialogs/keybackup/RestoreKeyBackupDialog.js]: oh yes, it is a bit - adjusted to just say it can't decrypt the backup

**Pull request merged**